--- origsrc/gimp-perl/Gimp/Makefile.PL	2000-07-20 09:44:16.000000000 -0500
+++ src/gimp-perl/Gimp/Makefile.PL	2007-04-22 12:36:50.484375000 -0500
@@ -4,7 +4,6 @@
 
 sub MY::const_config {
    my $self = shift;
-   $self->{LDDLFLAGS}="$GIMP_LIBS_NOUI $self->{LDDLFLAGS} $LDFLAGS $LIBS $INTLLIBS";
    package MY;
    $self->SUPER::const_config(@_);
 }
@@ -34,6 +33,7 @@
     'NAME'	=> 'Gimp::Lib',
     'VERSION_FROM' => '../Gimp.pm',
     'INC'	=> "$INC1 $GIMP_INC_NOUI $CPPFLAGS $pdl_inc $CFLAGS",
+    'LIBS'	=> "$GIMP_LIBS_NOUI $LDFLAGS $LIBS $INTLLIBS",
     'DEFINE'	=> "$DEFINE1 $DEFS",
     'macro'	=> { libdir => $libdir, exec_prefix => $exec_prefix, prefix => $prefix },
     'TYPEMAPS'	=> ["$topdir/typemap",@pdl_typemaps],
--- origsrc/gimp-perl/Makefile.PL	2005-09-19 22:27:33.000000000 -0500
+++ src/gimp-perl/Makefile.PL	2007-04-22 12:36:50.500000000 -0500
@@ -118,10 +118,10 @@
 	   $(INSTALLMAN1DIR) \
 	   $(INSTALLMAN3DIR) ; \
 	do \
-	   $(MKPATH) "$$dir"; \
-	   if test -d "$$dir" && test -w "$$dir" ; then : ; else \
+	   $(MKPATH) "$(DESTDIR)$$dir"; \
+	   if test -d "$(DESTDIR)$$dir" && test -w "$(DESTDIR)$$dir" ; then : ; else \
 	      echo ; \
-	      echo "ERROR: installation directory \"$$dir\"" ; \
+	      echo "ERROR: installation directory \"$(DESTDIR)$$dir\"" ; \
 	      echo "       is not writable, not installing gimp-perl" ; \
 	      echo ; \
 	      exit ; \
--- origsrc/gimp-perl/UI/Makefile.PL	2004-03-28 10:42:56.000000000 -0600
+++ src/gimp-perl/UI/Makefile.PL	2007-04-22 12:36:50.500000000 -0500
@@ -7,7 +7,6 @@
 
 sub MY::const_config {
    my $self = shift;
-   $self->{LDDLFLAGS}="$GIMP_LIBS $LDFLAGS $LIBS $self->{LDDLFLAGS}";
    package MY;
    $self->SUPER::const_config(@_);
 }
